.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js: node
+script: yarn run compress
+deploy:
+  provider: releases
+  api_key:
+    secure: TQuDynLxEXAfBJeC/dXpXhFzRt8a0DiYFQlfRoKnrAF5P7ZZg0MikEuvjXbwrKA2ArzXVBiFGYEnficjH9zv60ZDHddjDguB4wl72uHqR1lXhgMRzU3s4FTFdFv4g5H7iOOhrqYRh9Sxvgdvf260YxWAMKi8ip6QS1StTi0AEI1d56J+/95aDZZW8aWK9NFuQgZvcm1kBggIvewFKPrDIBEQEWSLeYGu2oT+hmv61s+t+RgUNXhjQ00VuWeHkGHMDzl4ljlJVlC1ZtrnqW9fiYs5VUeJZTX3Ob4e+7rd0noQTyGjaOReLrzuUWLr8MF2fZln9yuiZlhgkvLTmiEdCUgn3BcCFVfri3MgN9bBfuhNclSxVK6qluTPk4eHvPALZSZMjW+1tpPb1TIHa9ibYOfBX70mxSqlBCMjmTd2BGfkDVYFK6FdBLE1Ikc5Z7VQmYPFx8MBPdZBs95eJN6p8CqMhbM+bYHXFIqagXzWCxdps7in0xMUYuWkGjTc2QOA8KAjhorgyNomPY9cvdzDeZhKwFhSvJZQnPxQ3m4x2KMRGskJphTBA6VNwS9GktBq8F5KEQa59gewOcklRhf8kUzNy1QawU8YNrU28wUp8U9hb87NxQzJCJwQZBgMebg+j9EVoRh54jAZ4qNy6fHfBCm8w1XQQbLRLJBP+qZeQRY=
```

Comment:
So it's cool if the API key is public? I don't know about these things, it just "feels" odd to me.

Reply:
https://docs.travis-ci.com/user/encryption-keys/

Essentially, this is the encrypted value of the API Key - encrypted with a private key that only Travis has.

Reply:
The value, once decrypted, contains an API key that belongs to @umts-buildbot-gismap. It grants write access to public repositories which that user has write access to. And, that user only has write access to this repository.